### DISCUSSION TITLE
Register as secondary Device - did not work - Ubuntu Touch

### DISCUSSION THREAD

**Discussion by n00bsi** [axolotl-chat/axolotl#1413] [General]
Hi,
using Ubuntu Touch 20.04
and axolotl Version 2.0.5 from OpenStore.

If I scan the QR-Core from the Primary Device Signal Version: 7.19.2
nothing happen on the Ubuntu Touch only the QR-Code ist renewing after
clicking couple at the Primary Device.

How to get Logs to may see whats wrong ?

**Comment by Bu156** [comment]
I am facing the same issue with Ubuntu Touch 20.04 on Fairphone 4 using latest Molly (v7.21.5-2-FOSS) on Android and Axolotl from OpenStore.

**Comment by Bu156** [comment]
I managed to get it to work by downgrading my Signal mobile app (I used v7.6.2-2-FOSS of Molly) and was successfully able to get it to pair :)

**Comment by ruudfiers** [comment]
Problem unfortunately still exists for Ubuntu Touch. Is someone working on this?
